_component_design/radio-buttons.md

```diff
@@ -48,17 +38,23 @@ helix-ui-javascript: false
 
 <div class="hxRow" markdown="1">
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-12-md hxSpan-12-lg" %}  
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}  
 
-### Best practices
+## Best practices
 
-- Always use a label above the available options.
-- Keep the number of options to between two and five choices.
-- There is always a default option selected (preferably at the top or the list).
-- If the user clicks on another option, the first is unselected.
-- Radio buttons should only be used when the list can be vertical. If space is not available, consider a dropdown menu.
-- List options in an order that makes logical sense. Depending on the context, the order could be alphabetical or from most common to least common. For example, if listing accounts by name, list them alphabetically. In billing, paying the amount due should be listed above letting the user enter a custom payment amount.
-- Use checkboxes and radio buttons only to change settings, not as action buttons that make something happen.
+Consider the following best practices when you construct a radio button:
+
+- By default, the radio button at the top of the list is selected. The user can then make a different selection, if required.
+- When the user clicks a radio button, the previously selected button is deselected.
+- Keep the number of options to between two and five choices. If you’re providing more than five options, consider using a drop-down list.
+- List radio button options vertically, with one choice per line. If space is not available, consider using a drop-down list.
+- List radio button options in logical order. Depending on the context, the order could be alphabetical or from most common to least common. For example, if listing accounts by name, list them alphabetically. In billing, place the paying the amount due option above the enter a custom payment amount option.
+- Consider using a [switch]({{site.baseurl}}/components/switches.html) component if a singular input can be used to enable or disable a feature (in place of two yes/no radio buttons).
```

Comment:
@palindromebart, l left this best practice in because I didn't understand it. Can you help me understand what it might mean, or if we should remove it?